vissl/hooks/wandb_hook.py

```diff
@@ -0,0 +1,226 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import logging
+
+import torch
+from classy_vision import tasks
+from classy_vision.generic.distributed_util import is_primary
+from classy_vision.hooks.classy_hook import ClassyHook
+
+if is_primary():
+    import wandb
+
+BYTE_TO_MiB = 2 ** 20
+
+class SSLWandbHook(ClassyHook):
```

Comment:
A question on this: is this hook similar to Tensorboard hook with the only difference being in logging to "wandb" instead of tensorboard? 

Is it possible that we can inherit the `TensorboardHook` ? Or alternatively, does it make sense to extend the `TensorboardHook` directly to optionally log to wandb as well is user is using WandB ?